src/index.html

```diff
@@ -1,13 +1,80 @@
 <!doctype html>
+
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+    <link
+      rel="stylesheet"
+      href="./styles/index.scss"
+    >
   </head>
-  <body>
-    <h1>Product cards</h1>
+
+  <body class="page">
+    <div
+      class="page-cont"
+      data-qa="card"
+    >
```

Comment:
```suggestion
    <div class="page-cont" data-qa="card">
```

Reply:
Do you know how to set up VSCode to achieve that? It seems like all options are either one per line or all on the same line.
<img width="293" alt="Screenshot 2024-01-31 152044" src="https://github.com/mate-academy/layout_product-cards/assets/49961603/447efd14-f7c0-4852-94e0-a40146a2869f">

Reply:
not sure that vs code can do it, but you can change it manually for now and then try to find out with settings  later